assets/js/googlesitekit/widgets/datastore/widgets.js

```diff
@@ -44,7 +44,6 @@ const WidgetWidthKeys = Object.keys( WIDGET_WIDTHS )
 
 export const initialState = {
 	areaAssignments: {},
-	registryKey: undefined,
```

Comment:
This wasn't explicitly specced, but we might as well remove this unused property at the same time.